src/frontend/apps/impress/src/features/docs/doc-management/api/useRemoveFavoriteDoc.tsx

```diff
@@ -0,0 +1,59 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+
+import { APIError, errorCauses, fetchAPI } from '@/api';
+import { Doc, KEY_LIST_DOC } from '@/features/docs';
+
+export type RemoveFavoriteDocParams = Pick<Doc, 'id'>;
+
+export const removeFavoriteDoc = async ({
```

Comment:
Same I would have put delete, I know we have `useRemoveDoc` but I regret.
```suggestion
export const deleteFavoriteDoc = async ({
```